contracts/storage/OptionsVaultStorage.sol

```diff
@@ -25,7 +25,7 @@ contract OptionsVaultStorageV1 is
     address private _oldAsset;
 
     // Privileged role that is able to select the option terms (strike price, expiry) to short
-    address public manager;
+    address private _manager;
```

Comment:
do we need manager here?

Reply:
Trying to keep the storage layout compatible with v1 atm, so we have to retain it